app/src/main/res/layout/previous_responses_header_item.xml

```diff
@@ -15,28 +15,29 @@
     android:layout_height="wrap_content"
     android:clickable="true"
     android:focusable="true"
+    android:minHeight="@dimen/clickable_item_min_height"
     android:onClick="@{(v) -> viewModel.onResponsesHeaderClicked()}"
-    android:padding="0dp"
+    android:paddingTop="24dp"
```

Comment:
In this container I have removed `marginTop=24dp` and shifted it to `paddingTop=24dp` so that it can be the part of `minheight=48dp`.